client/src/Routes.js

```diff
@@ -18,6 +19,10 @@ const Routes = () => {
       <Route path="/login" component={Login} />
       <Route path="/applicant_create_profile" component={ApplicantProfile} />
       <Route path="/profile_creation_successful" component={SuccessMessage} />
+      <Route
```

Comment:
The routes on 20 and 21 need to be updated to use - to represent spaces to match the convention and so that they work when called from places where they have already been updated.